configuring project in CMake by -DAV_BUILD_PNG=OFF does not work

<!--
Please ask questions ("how do I...?") on the [alicevision@googlegroups.com mail list](http://groups.google.com/group/alicevision)
not by filing issues which are used for bugs and tasks organization.

Please add the label BUG if it is a bug report.
-->

### Problem
Hi all,

According to Install.md I should be able to configure the project
by CMake by disabling AV_BUILD_PNG. But in this case I get the
following error in CMake,

> CMake Error at C:/Program Files/CMake/share/cmake-3.9/Modules/ExternalProject.cmake:2295 (get_property):
>   get_property could not find TARGET png.  Perhaps it has not yet been
>   created.

I am using Windows 7, Visual Studio 2013 and CMake-gui.

Apparently another dependency in the project (openimageio) requires PNG
and that is making this issue.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -211,7 +211,7 @@ ExternalProject_Add(openimageio
        BINARY_DIR ${BUILD_DIR}/openimageio_build
        INSTALL_DIR ${BUILD_DIR}/openimageio_build
        CONFIGURE_COMMAND ${CMAKE_COMMAND} -DCMAKE_BUILD_TYPE=${DEPS_CMAKE_BUILD_TYPE} -DCMAKE_PREFIX_PATH="${BUILD_DIR}/png_build;${BUILD_DIR}/turbojpeg_build" -DBOOST_ROOT=${BUILD_DIR}/boost_build -DILMBASE_HOME=${BUILD_DIR}/openexr_build -DOPENEXR_HOME=${BUILD_DIR}/openexr_build -DTIFF_INCLUDE_DIR=${BUILD_DIR}/tiff_build/include -DTIFF_LIBRARIES=${BUILD_DIR}/tiff_build/lib/libtiff.so ${PNG_CMAKE_FLAGS} ${JPEG_CMAKE_FLAGS} -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR> <SOURCE_DIR> -DUSE_PYTHON=OFF -DUSE_OPENCV=OFF -DUSE_OPENGL=OFF
-       DEPENDS boost openexr tiff png ${JPEG_TARGET}
+       DEPENDS boost openexr tiff ${PNG_TARGET} ${JPEG_TARGET}
        )
 # TODO: Raw, openjpeg
 #  -DLibRaw_INCLUDE_DIR=$LIBRAW_INCLUDE_DIR

